src/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_library(asteroid-helloworld main.cpp resources.qrc)
+set_target_properties(asteroid-helloworld PROPERTIES PREFIX "")
+set_target_properties(asteroid-helloworld PROPERTIES SUFFIX "")
```

Comment:
You forgot this one here:
`set_target_properties(asteroid-helloworld PROPERTIES PREFIX "" SUFFIX "")`